packages/unmock-core/src/service/serviceStore.ts

```diff
@@ -25,20 +25,16 @@ export class ServiceStore {
   }
 
   public match(sreq: ISerializedRequest): MatcherResponse {
-    for (const service of Object.values(this.serviceMapping)) {
-      const matchResponse = service.match(sreq);
-      if (matchResponse !== undefined) {
-        return matchResponse;
-      }
-    }
-    return undefined;
+    return Object.values(this.serviceMapping)
+      .map(service => service.match(sreq))
+      .filter(res => res !== undefined)[0];
```

Comment:
Taking the first element of a possibly empty list looks very dangerous but I guess JS allows that so...For stuff like this the `Option` data type would be perfect.